lib/fastlane/plugin/wpmreleasetoolkit/actions/common/close_milestone_action.rb

```diff
@@ -9,11 +9,13 @@ class CloseMilestoneAction < Action
       def self.run(params)
         repository = params[:repository]
         milestone_title = params[:milestone]
+        access_token = params[:access_token]
 
-        milestone = Fastlane::Helper::GithubHelper.get_milestone(repository, milestone_title)
-        UI.user_error!("Milestone #{milestone_title} not found.") if milestone.nil?
+        github_helper = Fastlane::Helper::GithubHelper.new(github_token: access_token)
 
-        Fastlane::Helper::GithubHelper.github_client().update_milestone(repository, milestone[:number], state: 'closed')
+        milestone = github_helper.get_milestone(repository, milestone_title)
+        UI.user_error!("Milestone #{milestone_title} not found.") if milestone.nil?
+        github_helper.update_milestone(repository, milestone[:number], state: 'closed')
```

Comment:
`update_milestone` is a method of `Octokit::Client`, not `GithubHelper`.
(and sadly we never took the time to write unit tests / specs on those actions (boooo! 😅 ), hence why this issue wasn't highlighted by CI and would only wait us using it to crash at runtime…)

Reply:
Fixed 👍